app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Formatters.scala

```diff
@@ -24,14 +24,21 @@ import scala.util.control.Exception.nonFatalCatch
 
 trait Formatters {
 
-  private[mappings] def stringFormatter(errorKey: String, args: Seq[String] = Seq.empty): Formatter[String] =
+  private[mappings] def stringFormatter(
+    errorKey: String,
+    args: Seq[String] = Seq.empty,
+    sanitise: String => String
+  ): Formatter[String] =
     new Formatter[String] {
 
```

Comment:
Do we need a sanitise function here? Seems like we always use .trim for the stringFormatter?

Reply:
We do as in most cases we get rid of all whitespace. In some situations we only trim.